modules/nf-core/sourmash/taxannotate/main.nf

```diff
@@ -0,0 +1,43 @@
+process SOURMASH_TAXANNOTATE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::sourmash=4.6.1" : null)
```

Comment:
Note that the `enable_conda` parameter has been deprecated, please check against the `master` on nf-core/modules to change this declaration.

Reply:
Fixed